### PR TITLE
GP2-1338: Advice article page

### PR DESCRIPTION
**Pull request by stevejalim**
Wires up the ArticlePage to the template, including social media links and OpenGraph tags.

Doesn't refactor the model layer, but does tweak the CMS panel layout to make things a bit clearer

greatcms trade great_8020_admin_pages_140_edit_preview_ (1)

greatcms trade great_8020_advice_test-advice-article-listing-page_test-article-title_ (1)

greatcms trade great_8020_admin_pages_140_edit_

### Workflow

- [X] Ticket exists in Jira https://uktrade.atlassian.net/browse/GP2-1338
- [X] Jira ticket has the correct status.
- [X] [Changelog](CHANGELOG.md) entry added.

### Reviewing help

- [X] Includes screenshot(s) - ideally before and after, but at least after

### Housekeeping

- [ ] Added all new environment variables to Vault.
- [ ] Cleaned up old feature flags
- [ ] Upgraded any vulnerable dependencies.
- [ ] I have updated security dependencies
- [ ] Python requirements have been re-compiled.
- [ ] Frontend assets have been re-compiled.

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: domestic/templates/domestic/base.html [hunk 1]
```diff
@@ -97,17 +98,22 @@
...
-            {% if page.hero_image or page.hero_medium or page.hero_image_thumbnail or page.article_image_thumbnail %}
-                <meta property="og:image" content="{% firstof page.hero_image_thumbnail.url page.article_image_thumbnail.url page.hero_medium.url page.hero_image.url %}">
-            {% else %}
-                {% if request %}
-                <meta property="og:image" content="{% block og_image %}{% static_absolute 'directory_components/images/opengraph-image.png' %}{% endblock %}">
-                {% endif %}
+            {% comment %} There are a number of fields which might be workable as a sharing image {% endcomment %}
+            {% image page.hero_image width-480 as hero_image %}
+            {% image page.hero_medium width-480 as hero_image_medium %}
+            {% image page.article_image width-480 as article_image %}
```
Comment (stevejalim) [inline domestic/templates/domestic/base.html:113]: Anyone have a better suggestion than 480px max-width for what Open Graph images should be? @lewis-coulson-dit @jakehopking @dahfool ?

**Review by depsiatwal — APPROVED** [review]
LGTM
My only concerns is if we port everything into great.cms will it bloat this repo and make it too big :-( 

Also I notice additions to core.helpers I feel this may become too  large. Perhaps a logical split pay help.

**Comment by stevejalim** [comment]
@depsiatwal Yeah i hear you re bloating repo, but if they want it all in the same CMS, I think we're gonna have to live with it. Splitting out helpers if it grows is +1 from me, yeah. I've tried to keep the stuff from BAU domestic separate, in its own app, for the most part

**Comment by stevejalim** [comment]
@depsiatwal FWIW, for each Wagtail page ported to Magna, I'd ballpark it as needing about 30% of the overall code needed from d-cms and great-domestic, which is nice

**Pull request merged**